devops/Makefile

```diff
@@ -56,108 +58,60 @@ define DB_SERVICE_NAME
 "patroni-master"
 endef
 
-define FORMSFLOW_FORMS_NAME
-"forms-flow-forms"
-endef
-
-define FORMSFLOW_BPM_NAME
-"forms-flow-bpm"
-endef
-
-define FORMSFLOW_API_NAME
-"forms-flow-api"
-endef
-
-define FORMSFLOW_WEB_NAME
-"forms-flow-web"
+define FORMS_NAME
+"forms"
 endef
 
 define HAMONGO_DB_TEMPLATE_PATH
-"openshift/forms-flow-ai/database/mongo-ha.yml"
+"openshift/database/mongo-ha.yml"
 endef
 
 define HAMONGO_DB_PARAM_PATH
-"openshift/forms-flow-ai/database/mongo-ha-param.yml"
-endef
-
-define SECRETS_PATRONI_DB_TEMPLATE_PATH
-"openshift/forms-flow-ai/secrets/ffa-patroni-secrets.yml"
-endef
-
-define CREATE_FFA_DB_TEMPLATE_PATH
-"openshift/forms-flow-ai/database/create-ffa-db.yml"
-endef
-
-define FFA_BPM_DB_NAME
-"FFA_BPM_DB"
-endef
-
-define FFA_API_DB_NAME
-"FFA_API_DB"
+"openshift/database/mongo-ha-param.yml"
 endef
 
 define SECRETS_TEMPLATE_PATH
-"openshift/forms-flow-ai/secrets/secrets.yml"
+"openshift/secrets.yml"
 endef
 
 define SECRETS_PARAM_PATH
-"openshift/forms-flow-ai/secrets/secrets-param.yml"
+"openshift/secrets-param.yml"
 endef
 
-define FORM_AI_BUILD_TEMPLATE_PATH
-"openshift/forms-flow-ai/build.yml"
+define FORMS_BUILD_TEMPLATE_PATH
+"openshift/forms-build.yml"
 endef
 
-define FORM_AI_FORMS_DEPLOY_TEMPLATE_PATH
-"openshift/forms-flow-ai/forms-deploy.yml"
+define FORMS_DEPLOY_TEMPLATE_PATH
+"openshift/forms-deploy.yml"
 endef
 
-define FORM_AI_FORMS_URL
-"$(HOST_PREFIX)aest-ffa-forms.$(DOMAIN)"
+define FORMS_PARAM_PATH
+"openshift/forms-secrets.yml"
 endef
 
-define FORM_AI_BPM_DEPLOY_TEMPLATE_PATH
-"openshift/forms-flow-ai/bpm-deploy.yml"
+define FORMS_URL
+"$(HOST_PREFIX)aest-forms.$(DOMAIN)"
 endef
 
-define FORM_AI_BPM_URL
-"$(HOST_PREFIX)aest-ffa-bpm.$(DOMAIN)"
+define SOURCE_REPOSITORY_URL
+"https://github.com/bcgov/SIMS.git"
 endef
 
-define FORM_AI_API_DEPLOY_TEMPLATE_PATH
-"openshift/forms-flow-ai/api-deploy.yml"
+define FORMIO_SOURCE_REPO_URL
+"https://github.com/formio/formio.git"
 endef
 
-define FORM_AI_API_URL
-"$(HOST_PREFIX)aest-ffa-api.$(DOMAIN)"
-endef
-
-define FORM_AI_WEB_DEPLOY_TEMPLATE_PATH
-"openshift/forms-flow-ai/web-deploy.yml"
-endef
-
-define FORM_AI_WEB_URL
-"$(HOST_PREFIX)aest-ffa-serviceflow.$(DOMAIN)"
-endef
-
-define WEB_PARAM_PATH
-"openshift/forms-flow-ai/web-config.yml"
-endef
-
-define FORMSFLOW_SOURCE_REPO_URL
-"https://github.com/AOT-Technologies/forms-flow-ai.git"
-endef
-
-define FORMSFLOW_SOURCE_REPO_BRANCH
-"master"
+define FORMIO_SOURCE_REPO_TAG
+"v2.5.3"
 endef
 
 define SOURCE_REPOSITORY_REF
 "main"
 endef
 
 define SOURCE_CONTEXT_DIR
-"devops/forms-flow-ai/"
+"./sources/packages/"
```

Comment:
the `build-forms` is the only one using this, can we follow the same done for the others or change the others to use this?